tests/unit/test_addtoken.py

```diff
@@ -0,0 +1,10 @@
+from unittest import mock, TestCase
+from add_token import print_short
+
+class test_at(TestCase):
+    @mock.patch('builtins.print')
+    @mock.patch('add_token.get_network_info', return_value=['10.23.53.54', ['10.23.53.54'], str(32)])
+    def test_add_token(self, addtoken, mock_print):
+        # self.assertEqual(print_short('t', 'c'), 2)
+        print_short('t', 'c')
+        mock_print.assert_called_with('microk8s join 10.23.53.54:32/t/c')
```

Comment:
this is great, but only checks one trivial case. could you add (at least one) extra test with more IPs to be printed, where we verify the result?

Ideally, the tests should be a useful signal to know that we don't break the function in all the cases where it is use. If we only test one trivial input, then that signal might not be very trustworthy.

Reply:
Sure, I'll do that.

Reply:
Thanks!